src/App.js

```diff
@@ -2,16 +2,14 @@
 import React from 'react';
 import Calculator from './components/Calculator';
 
-class App extends React.Component {
-  render() {
-    const numbers = [7, 8, 9, 4, 5, 6, 1, 2, 3, 0];
-    const operations = ['÷', 'x', '-', '+', '='];
-    return (
-      <div>
-        <Calculator nums={numbers} ops={operations} />
-      </div>
-    );
-  }
-}
+const App = () => {
+  const numbers = [7, 8, 9, 4, 5, 6, 1, 2, 3, 0];
+  const operations = ['÷', 'x', '-', '+', '='];
```

Comment:
- [ ] It would be better to add the modulo "%" operation so your calculator is complete.

Reply:
I had the modulo but the calculator file provided by Micoverse listens for that divide sign. So it was either I change my operations or change the microverse file. I chose to change mine. 

I hope you understand my reasoning @medaminedev66